test.go

```diff
@@ -362,6 +363,14 @@ func (t *TwampTest) putMessageOnWire(padZero bool) (int, byte, time.Time) {
 	paddingSize := t.GetSession().config.Padding
 	padding := make([]byte, paddingSize, paddingSize)
 
+	// Note that Go initializes variables with zero-values, which in the case
+	// of a []byte happens to be a slice filled with zeros.
+	if !t.GetSession().GetConfig().ZeroPad {
+		if _, err := cRand.Read(padding); err != nil {
+			log.Fatalf("generating random padding: %s", err)
```

Comment:
if #22 is merged first, we will want to change this to return an error. otherwise, we will want to change this in #22.

Reply:
#22 was just merged. Can you update this pull request accordingly?

Reply:
done